<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTask</class>
 <widget class="QWidget" name="FormTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="task">
     <property name="text">
      <string>тут будет текст задачи</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="decision"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="data"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn1">
       <property name="text">
        <string>запустить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn2">
     <property name="text">
      <string>проверить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="answer"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="last_task">
       <property name="text">
        <string>Предыдущая задача</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_task">
       <property name="text">
        <string>Следующая задача</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back_button">
       <property name="text">
        <string>К списку задач</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
